apps/nextjs/utils/urlQuery.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Helpers for accessing values in context object of getServerSideProps.
+ *
+ * Next.js provides access to query params via context object, which
+ * contains the values as strings or string arrays. These helpers aim to
+ * reduce repetition in pages by returning the values in expected type.
+ */
+
+export type UrlQuery = string | string[] | undefined;
+
+/** Try returning the first query value, cast from a string to a boolean. */
+export const getBoolean = (val: UrlQuery): boolean | undefined => {
+  const stringVal = getString(val);
+
+  switch (stringVal?.toLowerCase()) {
```

Comment:
@MythicManiac The part that writes the URLs for backend requests (the data provider) only outputs one value, so caching of backend requests shouldn't be an issue.

I've no strong feelings about supporting multiple values on frontend-level. I'd wager that if we want to support only single value, it should be either `true` (for clarity) or `1` (for brevity)?

Reply:
I suppose it's fine as is, as long as the URL building is consistent. Choosing to not support other variants would be a nice way to enforce that at least.

If I were to choose, I'd have `true` & `false` as the supported values (`false` is needed in case the default is `true` somewhere)